source/extensions/quic_listeners/quiche/envoy_quic_client_session.cc

```diff
@@ -44,7 +44,12 @@ void EnvoyQuicClientSession::Initialize() {
 }
 
 void EnvoyQuicClientSession::OnCanWrite() {
+  uint64_t headers_to_send_old =
```

Comment:
mind going through and adding const to unchanged variables?

Reply:
done

Reply:
The two in this function can be const too, no?